docs/design/design_v1.10.md

```diff
@@ -1,6 +1,6 @@
 ## Implementation design for kubeadm
 
-<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/1.10/color/certified-kubernetes-1.10-color.png" align="right" width="150px">`kubeadm init` and `kubeadm join` together provides a nice user experience for creating a best-practice but bare Kubernetes cluster from scratch.
+<img src="https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/certified-kubernetes/1.10/color/certified-kubernetes-1.10-color.png" align="right" width="150px">`kubeadm init` and `kubeadm join` together provides a nice user experience for creating a best-practice but bare Kubernetes cluster from scratch.
```

Comment:
i think we should use https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/certified-kubernetes/1.14/color/certified-kubernetes-1.14-color.png here

or possibly the 1.15 image, but's not there yet.

Reply:
hm, actually this is in the 1.10 doc, so i guess the 1.10 image is fine.
/ok-to-test